force-app/main/default/classes/ContactTriggerUtility.cls

```diff
@@ -0,0 +1,26 @@
+public with sharing class ContactTriggerUtility {
+ 
+
+
+
+    public static void afterUpdate(List<Contact> newTrigger, Map<ID, Contact> oldTrigger){
+        System.debug('ContactTriggerUtility main class reached');
+        List<Account> allAccounts = [SELECT Id,Budget__c,Wallet__c,NumberOfBoughtCars__c,Profits_Of_Contacts__c FROM Account];
+        Map<ID, Account> allAccountsMap = new Map<ID, Account>();
+
+        for(Account account:allAccounts){
+                allAccountsMap.put(account.id, account);
+            }
```

Comment:
Jeżeli masz Query to możesz odrazu je zrzutować do Mapy bez takiego zabiegu, wystarczy że napiszesz 
Map<Id, Account> accountsMap = new Map<Id,Account>([select id from account where ...]);